### ISSUE TITLE
[Enhancement] Spray marks not being made when not inside tractor using Courseplay and Proseed while sowing.

### ISSUE THREAD

**Issue by TheDanishGamingTV** [Courseplay/Courseplay_FS22#2362]
**READ FIRST**
https://github.com/Courseplay/Courseplay_FS22/wiki

**Steps to reproduce**
Using courseplay and proseed while sowing at the same time and when getting to a row where you would have spray marks exit the tractor.

**Actual behavior**
What happens is that the sowingmachine doesnt make spray marks when your not sitting inside the tractor but when you enter the tractor it does.

**Expected behavior**
What we expected was that the sowingmachine would make spray marks regardless of the player being inside the tractor or not when using Courseplay and Proseed at the same time while sowing.

**Screenshots**
A video of the issue.
https://www.youtube.com/watch?v=-UObHDnz77Y

**Courseplay version**
Version used: 7.1.2.4

**Log**
Please attach a log with all debug channels active.
For help with the debug channels,
please check out the readme at the github front page.

**Map**
Bantikow

**Vehicles/ implements involved**
I dont really think that has anything to do with the issue at hand.

**Relevant script mods**
Courseplay and Proseed.

**Comment by Tensuko** [comment]
Becouse AI doesn't have fruit destructions.
Beside that: Version used: 7.1.2.4
a bit old version....

**Comment by Tensuko** [comment]
Keep in mind that if we make that possible, there is no way at any given time that we can make sure the sprayer will follow that tramlines exactly like it was created by the seeder!
If this is what ppl ask for, we won't add that option at all.

**Comment by TheDanishGamingTV** [comment]
I see what you mean.
Thansk for answering though :)

**Comment by Flicher** [comment]
I have made the test. 
I dont need the sprayer to follow the trimline. I will do that my self.  All I wanted was to have the CP make the trimeline.

**Comment by daantimmer** [comment]
> I have made the test. I dont need the sprayer to follow the trimline. I will do that my self. All I wanted was to have the CP make the trimeline.

This integration sounds interesting indeed.

**Comment by schwiti6190** [comment]
Added with version 7.2.1.7

**Comment by Flicher** [comment]
Hi there. Sorry for the repons after this long time, but I just got the need for the tramline with coureplay again. After not haven't used CP for a long time.

But I can't get CP to make tramline with proseed on auto mode. 
Semi mode works fine. It like it can't registers the GPS

But the thing i wanted when my friend made this request was that I can use 1-2 seeder, where one is driving by cp and the other by be.

I have tried with version 7.4.1.0 And version 7.2.1.7 where you said the had been added.
